setup.cake

```diff
@@ -14,9 +14,10 @@ BuildParameters.PrintParameters(Context);
 
 ToolSettings.SetToolSettings(context: Context,
                             dupFinderExcludePattern: new string[] { 
-                                BuildParameters.RootDirectoryPath + "/src/Cake.Http.Tests/**/*.cs" },
+                                BuildParameters.RootDirectoryPath + "/src/Cake.Http.Tests/**/*.cs",
+								BuildParameters.RootDirectoryPath + "/**/Cake.Http.AssemblyInfo.cs"},
```

Comment:
Because there is a tool which generate the Cake.Http.AssemblyInfo.cs for both net45 and netstandard1.6 and dupFinder tool doesn't like it :)